tests/performance_tests/performance_test_basic.py

```diff
@@ -78,6 +78,7 @@ class SpecifiedContract:
             contractDir: str = "unittests/contracts/eosio.system"
             wasmFile: str = "eosio.system.wasm"
             abiFile: str = "eosio.system.abi"
+            account: Account = Account(accountName)
```

Comment:
Should the keys be put into the account now?  Seems like `accountName`, `*PrivateKey`, `*PublicKey` fields could all go away and simply be set on the `Account` object.

Reply:
I think accountName is useful to save for certain checks in our code, even if later it becomes redundant with specifiedContract.account.name, the others we can definitely get rid of.

Reply:
Never mind, removed all of it https://github.com/AntelopeIO/leap/pull/713/commits/5a67397e1c584af844ba68d7188d8cf73361ee97

Reply:
Where are the keys being set now on the `account` in `SpecifiedContract`?